src/objects.js

```diff
@@ -1,42 +1,70 @@
 const createPerson = (name, age) => {
-  // your code here
+  const person = {name:name, age:age};
+  return person;
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  const keys = Object.keys(object);
+  for (let i = 0; i < keys.length; i++) {
+    if (keys[i] === property.toString()) {
+      return true;
+    }
+  }
+  return false;
```

Comment:
this can be shortened to `return property in object`. [you can read more about the `in` operator here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in)